client/opentracing/src/test/java/org/hawkular/apm/client/opentracing/APMTracerReferenceTest.java

```diff
@@ -50,6 +54,40 @@
     private static final String TEST_APM_TRACEID = "xyz";
 
     @Test
+    public void testServiceNameIsAddedToRootNode() {
```

Comment:
The test is good - but could you move it to APMTracerTest as this test class is more for testing the handling of multiple references.